views/files/files.hbs

```diff
@@ -69,11 +69,11 @@
 				</strong>
 				{{#userHasPermission "FOLDER_DELETE"}}
 				<div class="pull-right">
-					<a data-method="dir-rename" title="{{$t " files.label.renameFolder"}}" data-directory-id="{{../this._id}}"
+					<a data-method="dir-rename" title="{{$t "files.label.renameFolder"}}" data-directory-id="{{../this._id}}"
 						data-directory-name="{{../this.name}}" tabindex="0">
 						<i class="fa fa-edit directory-icon" aria-hidden="true"></i>
```

Comment:
for some reason there seems to occur an empty space area after this icon and before the next one, resulting in a weirdly looking underscore behind the "rename" button (check console than its easier to see as an element). maybe you can try to fix that "extra space" somehow as well while working on that line anyway :)